### PR TITLE
iscsi: set max_data_area_mb for tcmu devices

### PR DESCRIPTION
**Pull request by liulanzheng**
Signed-off-by: liulanzheng <lanzheng.liulz@alibaba-inc.com>

### CODE REVIEW

File: pkg/snapshot/iscsi.go [hunk 1]
```diff
@@ -50,6 +50,8 @@ const (
 	// Naa preffix for loopback devices in configfs
 	// for example snID 128, the loopback device config in /sys/kernel/config/target/loopback/naa.1990000000000128
 	obdLoopNaaPreffix = 199
+
+	obdMaxDataAreaMB = 4
```
Comment (fuweid) [inline pkg/snapshot/iscsi.go:54]: what is it used for?

Comment (liulanzheng) [reply pkg/snapshot/iscsi.go:54]: It is used for setting the max_data_area_mb param for tcmu devices, which is the mmap buffer size for I/O requests. The default max_data_area_mb value is 1024MB which is too big and may cause memory waste.

Comment (fuweid) [reply pkg/snapshot/iscsi.go:54]: could you add the comment for the variable? I think it is good for the reader.

Comment (liulanzheng) [reply pkg/snapshot/iscsi.go:56]: yes, a description is added.

**Review by fuweid — APPROVED** [review]

**Pull request merged**